src/auth/OIDCConnector/utils.ts

```diff
@@ -44,6 +44,7 @@ export async function logout(auth: AuthContextProps, bounce?: boolean) {
       key.startsWith(GLOBAL_FILTER_KEY)
   );
   deleteLocalStorageItems([...keys, OFFLINE_REDIRECT_STORAGE_KEY, LOGIN_SCOPES_STORAGE_KEY]);
+  // FIXME: Remove this one local preview is enabled by default
   const pathname = isBeta() ? getRouterBasename() : '';
```

Comment:
@Hyperkid123 I'm assuming once ready - we can delete the `isBeta` function to flush out any locations like this?

Reply:
Exactly. We will no longer need the `/preview` basename.